src/components/global/minifooter.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Image from 'next/image';
+import copy from '../../../public/images/copy.png';
+
+const MiniFooter = () => {
```

Comment:
Please see if you can make a common mini footer for all the pages. 

This leads to duplication of the text and component.

Reply:
Code is present here - https://github.com/jiffy-labs/jiffyscan-frontend/blob/development/src/components/global/footer/Footer.tsx#L63 

Try to make a common component and use it in both places. Let me know if there are any issues with doing that.